Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN apt update \
          gdal-bin libgdal-dev \
          libhdf5-dev libhdf5-103 \
          libblas-dev libblas3 liblapack-dev liblapack3 liblapack-doc \
-         python-is-python3 python3-numpy python3-matplotlib python3-scipy python3-shapely python3-packaging \
+         python-is-python3 python3-numpy python3-matplotlib python3-scipy python3-shapely \
```

Comment:
python3-packaging is called out as required in the GAMMA 20220630 installation guide, regardless of whether we're using Ubuntu 20.04 or 22.04